### DISCUSSION TITLE
openzeppelin file not importing

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#1780] [Q&A] [answered]
even though installing correctly and setting remappings in my wsl system showing below error
"Source "@openzeppelin/contracts/token/ERC20/ERC20.sol" not found: File not found. Searched the following locations: "".
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/d2b9ba53-27ba-41aa-9da6-772115b3855e)
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85685981/4846998c-2843-47ee-8b2e-6b89c9d14ad8)

**Comment by urosognjenovic** [comment]
Try adding this one:

```
@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/
```

**Reply by panditdhamdhere** [reply]
tried it aint working thank you very much sir

**Comment by EngrPips** [comment]
Hello @panditdhamdhere, The file's destination might have changed in the version of the library you have installed now, so I suggest you search for the file manually and get the file's location. With that You can either do the native importing, where you explicitly specify the path, or you can use the location to create a remapping that would work appropriately. You can search for files on Mac with `cmd + p`, and it should be `cltr + p` on Windows.

**Comment by urosognjenovic** [comment, marked as answer]
Adding the line ```@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/``` to the ```remappings.txt``` file solved the problem for me.

**Reply by panditdhamdhere** [reply]
Thank you it worked

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.

**Reply by urosognjenovic** [reply]
I'm glad it worked for you! Sometimes just reopening VSCode after getting an import error works for me. I don't know why though, could be due to VSCode not automatically importing the remappings?
